src/components/companion/blueprintProfiles/attendeeInfo.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { AnimatedBorder } from '../../ui/animated-border';
+
+interface UserProfile {
+    name: string;
+    role: string;
+    hobby: string;
+    linkedIn: string;
+    funFacts: string[];
+    interests: string[];
+    additionalLinks: string[];
+    profilePicUrl: string;
```

Comment:
Note that some people haven't attached profile pictures - you may need to use Izzy's solution of defaulting to initials. You can bring in Izzy's solution later - for now in this PR, let's make profile pic an optional field.

Reply:
Yes, we'll be choosing images